src/instructlab/sdg/utilblocks.py

```diff
@@ -95,3 +95,68 @@ def generate(self, samples: Dataset) -> Dataset:
             self.separator,
             self.ctx.dataset_num_procs,
         )
+
+
+class FlattenColumnsBlock(Block):
+    def __init__(
+        self, ctx, pipe, block_name: str, var_cols: list, value_name: str, var_name: str
+    ) -> None:
```

Comment:
Can you add a docstring for this one?

Reply:
you marked this resolved, but i don't see the docstring?

Reply:
Opps sorry I didn't look closely enough, I added 4 docstrings to the tests, I thought that was where you asked for them, will fix.